sae/trainer.py

```diff
@@ -217,10 +225,13 @@ def hook(module: nn.Module, _, outputs):
                     handle.remove()
 
             if self.cfg.distribute_modules:
-                hidden_dict = self.scatter_hiddens(hidden_dict)
+                input_dict = self.scatter_hiddens(input_dict)
+                output_dict = self.scatter_hiddens(output_dict)
 
-            for name, hiddens in hidden_dict.items():
-                raw = self.saes[name]  # 'raw' never has a DDP wrapper
+            for name, outputs in output_dict.items():
+                # 'inputs' is distinct from outputs iff we're transcoding
```

Comment:
```suggestion
                # 'inputs' is distinct from outputs if we're transcoding
```

Reply:
oh I actually meant to use iff = if and only if lol

Reply:
TIL